tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -6580,6 +6494,20 @@ static TableParameter PackUnpackOpParameters[] = {
     {L"Validation.Input", TableParameter::UINT32_TABLE, true},
 };
 
+static TableParameter LongVectorBinaryOpParameters[] = {
+    {L"DataType", TableParameter::STRING, true},
+    {L"OpTypeEnum", TableParameter::STRING, true},
+    {L"InputValueSetName1", TableParameter::STRING, false},
+    {L"InputValueSetName2", TableParameter::STRING, false},
+};
+
+static TableParameter LongVectorUnaryOpParameters[] = {
+    {L"DataType", TableParameter::STRING, true},
+    {L"OpTypeEnum", TableParameter::STRING, true},
+    {L"InputValueSetName1", TableParameter::STRING, false},
+    {L"InputArgsName", TableParameter::STRING, false},
```

Comment:
Surprised BinaryOpParameters doesn't need something like InputArgsName.

Reply:
It will eventually. Non of the binary intrinsics added up to this point have required input arguments. Although, there might be a better name to use here. The 'InputArgNames' are used for additional arguments like the min/max values in clamp. Distinct from the unary, binary, or ternary operands.